jina/optimizers/__init__.py

```diff
@@ -105,22 +107,24 @@ class FlowOptimizer(JAMLCompatible):
     """
 
     def __init__(
-        self,
-        flow_runner: 'FlowRunner',
-        parameter_yaml: str,
-        evaluation_callback: 'OptimizerCallback',
-        n_trials: int,
-        workspace_base_dir: str = '',
-        sampler: str = 'TPESampler',
-        direction: str = 'maximize',
-        seed: int = 42,
+            self,
+            flow_runner: 'FlowRunner',
+            parameter_yaml: str,
+            evaluation_callback: 'OptimizerCallback',
+            n_trials: int,
+            workspace_base_dir: str = '',
+            output_file: Optional[str] = None,
```

Comment:
```suggestion
            output_dir: Optional[str] = None,
```

Reply:
good finding. It has to be `output_file`. The docstring was wrong.